app/src/main/java/com/vmware/nimbus/ui/main/OptionsActivity.java

```diff
@@ -47,6 +55,16 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean isChecked) {
             }
         });
 
+        loginSource.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+            @Override
+            public void onCheckedChanged(CompoundButton compoundButton, boolean isChecked) {
```

Comment:
With new login toggle, Saved/Restored messages should be clearer that they are saving/restoring the URI specifically.